aspnetcore/blazor/performance/app-download-size.md

```diff
@@ -0,0 +1,108 @@
+
+## Compression
+
+*This section only applies to Blazor WebAssembly apps.*
+
+When a Blazor WebAssembly app is published, the output is statically compressed during publish to reduce the app's size and remove the overhead for runtime compression. Blazor relies on the server to perform content negotiation and serve statically-compressed files.
+
+After an app is deployed, verify that the app serves compressed files. Inspect the **Network** tab in a browser's [developer tools](https://developer.mozilla.org/docs/Glossary/Developer_Tools) and verify that the files are served with `Content-Encoding: br` (Brotli compression) or `Content-Encoding: gz` (Gzip compression). If the host isn't serving compressed files, follow the instructions in <xref:blazor/host-and-deploy/webassembly/index#compression>.
+
+## Disable unused features
+
+*This section only applies to client-side Blazor scenarios.*
+
+Blazor WebAssembly's runtime includes the following .NET features that can be disabled for a smaller payload size.
+
+Blazor WebAssembly carries globalization resources required to display values, such as dates and currency, in the user's culture. If the app doesn't require localization, you may [configure the app to support the invariant culture](xref:blazor/globalization-localization#invariant-globalization), which is based on the `en-US` culture.
+
+:::moniker range=">= aspnetcore-8.0"
+
+Adopting [invariant globalization](xref:blazor/globalization-localization#invariant-globalization) only results in using non-localized timezone names. To trim timezone code and data from the app, apply the `<InvariantTimezone>` MSBuild property with a value of `true` in the app's project file:
+
+```xml
+<PropertyGroup>
+<InvariantTimezone>true</InvariantTimezone>
+</PropertyGroup>
+```
+
+> [!NOTE]
+> [`<BlazorEnableTimeZoneSupport>`](xref:blazor/performance/app-download-size#disable-unused-features) overrides an earlier `<InvariantTimezone>` setting. We recommend removing the `<BlazorEnableTimeZoneSupport>` setting.
+
+:::moniker-end
+
+:::moniker range="< aspnetcore-8.0"
+
+A data file is included to make timezone information correct. If the app doesn't require this feature, consider disabling it by setting the `<BlazorEnableTimeZoneSupport>` MSBuild property to `false` in the app's project file:
+
+```xml
+<PropertyGroup>
+<BlazorEnableTimeZoneSupport>false</BlazorEnableTimeZoneSupport>
+</PropertyGroup>
+```
+
+:::moniker-end
+
+:::moniker range="< aspnetcore-5.0"
+
+Collation information is included to make APIs such as <xref:System.StringComparison.InvariantCultureIgnoreCase?displayProperty=nameWithType> work correctly. If you're certain that the app doesn't require the collation data, consider disabling it by setting the `BlazorWebAssemblyPreserveCollationData` MSBuild property in the app's project file to `false`:
+
+```xml
+<PropertyGroup>
+<BlazorWebAssemblyPreserveCollationData>false</BlazorWebAssemblyPreserveCollationData>
+</PropertyGroup>
+```
+
```

Comment:
Please add link to https://github.com/dotnet/runtime/blob/main/src/mono/wasm/features.md for more flags